apps/src/code-studio/components/WorkspaceAlert.jsx

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Alert from '@cdo/apps/templates/alert';
+import $ from 'jquery';
+
+export default class WorkspaceAlert extends React.Component {
+  static propTypes = {
+    type: PropTypes.oneOf(['error', 'warning', 'notification']).isRequired,
+    children: PropTypes.element,
+    onClose: PropTypes.func,
+    isBlockly: PropTypes.bool,
+    isCraft: PropTypes.bool,
+    displayBottom: PropTypes.bool
+  };
+
+  render() {
+    const {displayBottom} = this.props;
```

Comment:
nit: get all props here i.e.
const {displayBottom, isBlockly, isCraft} = this.props;